w3af/plugins/crawl/url_fuzzer.py

```diff
@@ -41,19 +41,30 @@ class url_fuzzer(CrawlPlugin):
     Try to find backups, and other related files.
     :author: Andres Riancho (andres.riancho@gmail.com)
     """
-    _appendables = ('~', '.tar.gz', '.gz', '.7z', '.cab', '.tgz',
-                    '.gzip', '.bzip2', '.inc', '.zip', '.rar', '.jar', '.java',
-                    '.class', '.properties', '.bak', '.bak1', '.bkp', '.back',
-                    '.backup', '.backup1', '.old', '.old1', '.$$$'
+    _appendables = ('~', '~~', '_', '.', '.tar.gz', '.gz', '.7z', '.cab',
```

Comment:
Now that these lists are getting larger, we have the risk of duplicating one item in them.

Let's change the type from tuple (current) to set (desired). By doing so we make sure that even if we add duplicates in the variable definition, the set will eliminate them.

Just change: `_appendables = (...)` with `_appendables = {...}`.

The same change should be applied to all other class attributes that define lists like this in the url_fuzzer class.

Reply:
Done.